### ISSUE TITLE
PHP strlen(): Argument #1 ($string) must be of type string, array given

### ISSUE THREAD

**Issue by mdobydullah** [ccxt/ccxt#13828]
Hi,

```
CCXT version: 1.87.11
PHP version: 8.1.7
Laravel version: 9.17.0
Exchange: binance
```

I've updated ccxt version to **1.87.11** and faced the issue. I tried to create order and faced the issue.

```php
$order = $exchange->create_order('XRP/USDT', 'market', 'buy', 500);
```

```php
strlen(): Argument #1 ($string) must be of type string, array given

\vendor\ccxt\ccxt\php\Exchange.php line number: 2548

if (gettype($trades) === 'array' && array_keys($trades) === array_keys(array_keys($trades)) && strlen($trades)) {
```

Everything was okay in ccxt **1.83.65**.

Please help me to resolve the issue. Thanks in advance.

**Comment by kroitor** [comment]
@mdobydullah thank you very much for reporting it! We've pushed a fix in CCXT 1.87.21, the new build will arrive in ~20 minutes. Please, let us know if that solves the issue for you or not.

**Comment by mdobydullah** [comment]
@kroitor I've just tested. It works perfectly. Thanks for fixing the bug so quickly. I really appreciate it! 👌
